bionemo-recipes/README.md

```diff
@@ -123,7 +123,8 @@ ______________________________________________________________________
 
 We prioritize **readability and simplicity** over comprehensive feature coverage:
 
-- **KISS over DRY**: It's better to have clear, duplicated code than complex abstractions
+- **KISS (Keep It Simple) over DRY (Don't Repeat Yourself)**: It's better to have clear, duplicated code than complex
```

Comment:
I still find these a little odd, because I haven't seen these in the wild so much. But if you think people will be familiar, I'm okay to leave it as is.